src/Instrument/Transformer/CachingTransformer.php

```diff
@@ -87,10 +96,11 @@ public function transform(StreamMetaData $metadata)
         }
 
         $originalUri = $metadata->uri;
-        $cacheUri    = str_replace($this->rootPath, $this->cachePath, $originalUri);
+        $cacheUri    = $this->kernel->getCachePathResolver()->getCachePathForResource($originalUri);
 
         $lastModified  = filemtime($originalUri);
-        $cacheModified = file_exists($cacheUri) ? filemtime($cacheUri) : 0;
+        $cacheExisted = file_exists($cacheUri);
```

Comment:
I'll think a little about this (on my way to the home now and preparing for podcasting)